conf/development-app-sentry.ini

```diff
@@ -0,0 +1,109 @@
+[pipeline:main]
```

Comment:
Was thinking of keeping it around to save pain if this needs to be debugged again, but you're right, probably by the time we need this file again it'll have diverged from the normal `development` config file.